patch-package
--- a/node_modules/inflight/inflight.js
+++ b/node_modules/inflight/inflight.js
@@ -1,6 +1,5 @@
 var wrappy = require('wrappy')
 var reqs = Object.create(null)
-var once = require('once')
 
 module.exports = wrappy(inflight)
 
@@ -8,7 +7,7 @@
   if (reqs[key]) {
     reqs[key].push(cb)
     return null
-  } else {
+  }
  else {
     reqs[key] = [cb]
     return maker(key)
@@ -16,7 +15,7 @@
 }
 
 function maker (key) {
-  return once(function FIRST () {
+  return function FIRST () {
     var cbs = reqs[key]
     var len = cbs.length
     var args = slice(arguments)
@@ -29,7 +28,7 @@
       }
       cbs[i].apply(null, args)
     }
-  })
+  }
 }
 
 function slice (args) {
